### ISSUE TITLE
 Chronograf (1.8.0) a.author is undefined

### ISSUE THREAD

**Issue by Lamera** [influxdata/chronograf#5408]
What browser are you using (name and version)?
Firefox 68.5.0esr (64-Bit)

What operating system are you using?
Debian 10 (Buster)

Please describe what you were trying to do when you encountered this error:
Under News Feed following text appears on a fresh installation of chronograf:
A Chronograf error has occurred. Please report the issue here.

```
l/<@http://192.168.1.30:8888/src.5692061f.js:2365:376
l@http://192.168.1.30:8888/src.5692061f.js:2365:310
sa@http://192.168.1.30:8888/src.5692061f.js:717:55869
$a@http://192.168.1.30:8888/src.5692061f.js:717:68127
oo@http://192.168.1.30:8888/src.5692061f.js:717:91055
uo@http://192.168.1.30:8888/src.5692061f.js:717:91442
Yo@http://192.168.1.30:8888/src.5692061f.js:717:98524
$o@http://192.168.1.30:8888/src.5692061f.js:717:97904
eu@http://192.168.1.30:8888/src.5692061f.js:717:99279
Mn@http://192.168.1.30:8888/src.5692061f.js:717:30665

```

**Comment by russorat** [comment]
@Lamera thanks for opening. We will investigate. If you have a chance, could you add a screenshot of the issue?

**Comment by Lamera** [comment]
@russorat Thank you for your short response. Here is the screenshot.
![grafik](https://user-images.githubusercontent.com/19840658/75716435-750d1a00-5ccf-11ea-8a96-f87ecb5a3fa6.png)

**Comment by glinton** [comment]
This should be fixed by #5420 and will be available in 1.8.1 (or is available immediately with a [nightly build](https://portal.influxdata.com/downloads/)). Comment if that is not the case and we'll re-open this)
